src/v2/es-utils.js

```diff
@@ -139,14 +160,23 @@ class ESClass {
       for (const index of Object.keys(importJson)) {
         const item = importJson[index];
         const _source = item._source;
-
-        if (item._type === 'index-pattern' && item._id === 'cicd-perf*') {
-          const apiHost = (this.env.HTTP_PORT !== 80) ? this.env.HOST + ':' + this.env.HTTP_PORT : this.env.HOST;
-          _source.fieldFormatMap = _source.fieldFormatMap.replace('__api__hostname', apiHost.toLowerCase());
+        if (_source === 'delete_this') {
```

Comment:
I just saw this, re: the references in `.kibana_items.json`. I think this would warrant big loud documentation. Not only could it potentially be a useful feature for the end users, but unless I understand this wrong, it's deleting items from Kibana, which end users may have already customized. So deleting them could produce unexpected and/or undesirable results.

Reply:
You're right and i added this because I discovered that i distributed a few items in earlier releases that didn't make any sense for anybody - it may be the first and last time that it happens. For the other items, I check if things already exist and if so, don't overwrite them